openmetadata-ui/src/main/resources/ui/src/components/TestSuitePipelineTab/TestSuitePipelineTab.component.tsx

```diff
@@ -550,13 +553,44 @@ const TestSuitePipelineTab = () => {
     currTriggerId,
   ]);
 
+  const errorPlaceholder = useMemo(
+    () => (
+      <ErrorPlaceHolder
+        button={
+          <Button
+            ghost
+            className="p-x-lg"
+            data-testid="add-placeholder-button"
+            icon={<PlusOutlined />}
+            type="primary"
+            onClick={() => {
+              history.push(getTestSuiteIngestionPath(testSuiteFQN));
+            }}>
+            {t('label.add')}
+          </Button>
+        }
+        className="mt-24"
+        heading={t('label.pipeline')}
+        permission={createPermission}
+        type={ERROR_PLACEHOLDER_TYPE.ASSIGN}
+      />
+    ),
+    [testSuiteFQN]
+  );
+
   if (isLoading || isFetchingStatus) {
     return <Loader />;
   }
 
-  return !isAirflowAvailable ? (
-    <ErrorPlaceHolderIngestion />
-  ) : (
+  if (!isAirflowAvailable) {
+    return <ErrorPlaceHolderIngestion />;
+  }
+
+  if (isEmpty(testSuitePipelines) && !(isLoading || isFetchingStatus)) {
```

Comment:
&& !(isLoading || isFetchingStatus)

This condition we already covers in above not sure how this helps, Can you check do we actually need this